samples/adapter-tyntec-whatsapp/.env

```diff
@@ -0,0 +1,2 @@
+TyntecApikey=
+Waba=
```

Comment:
What is Waba?

Reply:
Oh, I forgot to document the environment variables. I'll document them in the README.